substrate/primitives/core/src/const_hex2array.rs

```diff
@@ -15,8 +15,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Provides const function allowing converting hex string to `u8` array at compile time if used in
-//! proper context.
+//! Provides a const function for converting a hex string to a `u8` array at compile time, when used
+//! in the proper context. Valid characters are `[0-9a-fA-F]`, and the hex string should not start
+//! with the `0x` prefix.
+//!
+//! # Panics
```

Comment:
The panics section should be in the function doc

Reply:
done: ab13181